.gitignore

```diff
@@ -1,4 +1,3 @@
-*.py[co]
```

Comment:
This line is a duplicate. Even after this change, `*.pyc` will continue to be ignored :-)